version.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
I went back and forth, but ultimately put it in main because I was thinking if it as the global version for pick, not just the CLI version.

However, since this is only a cli tool, perhaps we should move that into commands.